### PR TITLE
Add webui_url to FrameworkInfo

### PR DESCRIPTION
**Pull request by guenter**
Tests pass, and tested setting the commandline option.

### CODE REVIEW

File: src/main/scala/org/apache/mesos/chronos/scheduler/config/MainModule.scala [hunk 1]
```diff
@@ -48,17 +50,29 @@ class MainModule(val config: SchedulerConfiguration) extends AbstractModule {
...
-    val frameworkInfo = FrameworkInfo.newBuilder()
+    val frameworkInfoBuilder = FrameworkInfo.newBuilder()
       .setName(config.mesosFrameworkName())
       .setCheckpoint(config.mesosCheckpoint())
       .setRole(config.mesosRole())
       .setFailoverTimeout(config.failoverTimeoutSeconds())
       .setUser(config.user())
-    frameworkIdUtil.setIdIfExists(frameworkInfo)
-    frameworkInfo.build()
+
```
Comment (elingg) [inline src/main/scala/org/apache/mesos/chronos/scheduler/config/MainModule.scala]: This is a nit, but let's but the else if and else should be one line up. See another example in the Chronos code:

//If the job was deleted after the taskId was added to the queue, the task could be empty.
      if (jobOption.isEmpty) {
        //remove invalid task
        removeTask(taskId)
        None
      } else if (jobOption.get.disabled) {
        None
      } else {
        Some(taskId, job)
      }

File: pom.xml [hunk 2]
```diff
@@ -31,7 +31,7 @@
     <joda-convert.version>1.7</joda-convert.version>
     <joda-time.version>2.3</joda-time.version>
     <lz4.version>1.2.0</lz4.version>
-    <mesos.version>0.20.1</mesos.version>
+    <mesos.version>0.21.0</mesos.version>
```
Comment (elingg) [inline pom.xml]: Nice

**Comment by elingg** [comment]
Just one style comment, but it looks great. :+1: Thanks for adding the web UI! Any comments @brndnmtthws?

**Comment by brndnmtthws** [comment]
:+1:, merge away.

**Comment by guenter** [comment]
@brndnmtthws @elingg thanks! Made the style change.

**Pull request merged**